pkg/json.go

```diff
@@ -30,6 +30,7 @@ import (
 type jsonCheckResult struct {
 	Name       string
 	Details    []string
+	ID         uint
 	Confidence int
 	Pass       bool
 }
```

Comment:
we shouldn't touch the old json format at this point in my opinion

Reply:
Removed